spec/integration/connect-spec.coffee

```diff
@@ -41,19 +44,45 @@ describe "Connecting to rdebug-ide", ->
       workspaceElement.querySelector('.ruby-debugger .connect').textContent.trim() == 'Disconnect'
 
     runs ->
-      # 6. Start button should now be active
+      # 6. button should now be active
       expect(workspaceElement.querySelector(buttonSelector)).not.toBeDisabled()
 
+  testStepButton = (buttonSelector) ->
+    connect()
+
+    runs ->
+      expect(workspaceElement.querySelector(buttonSelector)).toBeDisabled()
+      workspaceElement.querySelector('.ruby-debugger .connect').click()
+
+    waitsFor ->
+      workspaceElement.querySelector('.ruby-debugger .connect').textContent.trim() == 'Disconnect'
+
+    runs ->
+      workspaceElement.querySelector('.ruby-debugger .play').click()
+
+    waitsFor ->
+      workspaceElement.querySelector('.ruby-debugger .play').title.trim() == "Pause"
+
+    runs ->
+      workspaceElement.querySelector('.ruby-debugger .play').click()
```

Comment:
1. Because the tests are so difficult to read and understand what they do, I appreciate the step-comments. For example: It would be helpful if this line said `# X. Pause the debugger`.

2. IMO there doesn't need to be many integration tests. 1 per feature is enough. For example: instead of testing each button it could be made into 1 test asserting that the **pause (or breakpoint)** feature works.